### ISSUE TITLE
Cannot connect to telegram api on ipv6-only hosts

### ISSUE THREAD

**Issue by syndicut** [python-telegram-bot/python-telegram-bot#720]
### Steps to reproduce
1. Install latest version on ipv6-only host
2. Try connecting 

### Expected behaviour
Succesful connection

### Actual behaviour
NetworkError is throwed. Mostly of this commit, I think: https://github.com/python-telegram-bot/urllib3/commit/d0a67600f464399d023848e16d985e97522313c4

### Configuration
**Operating System:**
Reproducible on debian jessie/ubuntu xenial

**Version of Python, python-telegram-bot & dependencies:**
python 3.4
All versions >5.3.1 are affected, I believe

### Logs
```
NetworkError: urllib3 HTTPError HTTPSConnectionPool(host='api.telegram.org', port=443): Max retries exceeded with url: /bot326003279:MY_TOKEN_XXX/sendMessage (Caused by NewConnectionError('<telegram.vendor.ptb_urllib3.urllib3.connection.VerifiedHTTPSConnection object at 0x7f6770459750>: Failed to establish a new connection: [Errno -2] Name or service not known',))
```

**Comment by bomjacob** [comment]
Thanks for identifying the problem. This commit should have never found its way to release. It was for internal testing and playing with the unitest framework.
-- @tsnoam

**Comment by rahiel** [comment]
This should be fixed in version 7.0 that we'll be releasing soon.
